pom.xml

```diff
@@ -28,6 +35,10 @@
         <module>lighty-modules</module>
         <module>lighty-resources</module>
         <module>lighty-tests-report</module>
+        <module>bnd-parent</module>
```

Comment:
probably not needed as we will be using gnmi from ODL, from now build in karaf should do it all